consensus/dummy/test_consensus.go

```diff
@@ -0,0 +1,53 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
```

Comment:
We shouldn't need to migrate this in. We just pass in non-nil callbacks to the tests in `core/test_blockchain.go` to catch any errors that could come from mishandling extra state changes. Since those don't exist in subnet-evm, I think we can remove this file and pass in the normal dummy engine with no callbacks.

Reply:
I added this specifically for https://github.com/ava-labs/subnet-evm/blob/98737f6a870ffb3785b9acb5553eabee95ed787d/core/blockchain_test.go#L256. In this test we need to set the state so that pruner can work. I don't know if there is any other way but seems like this test breaks without setting the state in callbacks.